### DISCUSSION TITLE
如何获取特定适配器的事件？

### DISCUSSION THREAD

**Discussion by bsdayo** [koishijs/koishi#559] [Q&A] [answered]
比如qq上有人在群里戳一戳bot，怎么获取到这个事件然后让bot作出反应（

**Comment by idlist** [comment, marked as answer]
有个事件 `notice/poke`，不是稳定接口所以不在文档里

举个例子： https://github.com/idlist/2bot-v4/blob/main/plugins/fun/poke.js

还有几个以 `notice` 开头的相关事件，可以用 TS Typings 看一眼或者直接用 GitHub 搜

**Reply by bsdayo** [reply]
感谢，去看看

**Reply by DGCK81LNN** [reply]
[我也写了个回应戳一戳](https://github.com/DGCK81LNN/lnnbot/blob/f5ff8ff122677d6990ab59b6c2f104f19ae54dce/plugins/wassup.ts#L48-L51)呢…）

（ps. 私聊戳bot的时候，`session.targetId` 好像是没有值的……？
